optuna/pruners/hyperband.py

```diff
@@ -49,18 +47,39 @@ class HyperbandPruner(BasePruner):
         Thus, for example, if ``HyperbandPruner`` has :math:`4` pruners in it,
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
+    .. note::
+        Hyperband has several :class:`~optuna.pruners.SuccessiveHalvingPruner`. Each
+        :class:`~optuna.pruners.SuccessiveHalvingPruner` is referred as "bracket" in the original
+        paper. The number of brackets is an important factor to control the early stopping behavior
+        of Hyperband and is determined by ``max_resource`` and ``reduction_factor`` as
+        `n_brackets = floor(log(max_resource) / log(reduction_factor))`. Please set
+        ``max_resource`` and ``reduction_factor`` so that the number of brackets is not too large
+        (about 4 ~ 6 in most use cases).
+        Please see Section 3.6 of the original paper
+        <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ for the detail.
```

Comment:
This URL is not rendered correctly. Could you verify?

Reply:
I forgot to insert a back quote "`". Sorry for confusing.

Reply:
Thanks!